addons/links/package.json

```diff
@@ -33,6 +32,7 @@
     "shelljs": "^0.7.8"
   },
   "peerDependencies": {
+    "@storybook/addons": "^3.3.0-alpha",
```

Comment:
should this be `^3.3.0-alpha.3` ?

Reply:
It depends on whether we want to allow different versions of our packages to be used. If not, we should omit the caret in all cross-dependencies (direct and peer)

Reply:
Ok, it turns out that lerna can update peerDeps as well, so I changed the version to the current one everywhere